public/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
@rafayet-monon Should we update this file to have just the only thing you need and its message about your project?
Because you're using this one which is not related with your project anymore.
```
"short_name": "React App",
"name": "Create React App Sample",
```